backend/apps/sandbox/tests_triage.py

```
@@ -0,0 +1,206 @@
+            "submitted_response": "",
+        }
+        response = self.client.post(
+            f"/api/sandbox/triage-issues/{self.issue.id}/submit_triage/",
+            data=payload,
+            format="json",
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_my_attempts_returns_only_mine(self):
+        # Create an attempt for another user
+        other_user = User.objects.create_user(username="other_triager", password="pass")
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)